### PR TITLE
PCC15

### PR DESCRIPTION
**Pull request by code-monk08**
ATTENTION: before clicking "Create Pull Request" please submit some meta data, thanks!

Difficulty level (1-10): [2]
Estimated time spent (hours): [4-5]
Completed (yes/no): [yes]
I stretched my coding skills (if yes what did you learn?): [Flask]
Other feedback (what can we improve?): [All Good !]

### CODE REVIEW

File: 15/code-monk08/app.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+#run export FLASK_ENV=development before running flask run
+import datetime, requests
+from flask import Flask, render_template, request, redirect
+from flask_sqlalchemy import SQLAlchemy
+
+key = open("apikey.txt","r").readline()
```
Comment (bbelderbos) [inline 15/code-monk08/app.py]: use a with statement?

File: 15/code-monk08/templates/weather.html [hunk 2]
```diff
@@ -0,0 +1,251 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Simple Weather App</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <style>
```
Comment (bbelderbos) [inline 15/code-monk08/templates/weather.html]: I would put styles in its own file

File: 15/code-monk08/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Simple Weather App
+
```
Comment (bbelderbos) [inline 15/code-monk08/README.md]: nice you included a readme!

**Comment by bbelderbos** [comment]
Great work @code-monk08

File: 15/code-monk08/app.py (hunk 1 above)
Comment (code-monk08) [reply 15/code-monk08/app.py]: yes, I use with statement these days.. PR is from April 12

File: 15/code-monk08/templates/weather.html (hunk 2 above)
Comment (code-monk08) [reply 15/code-monk08/templates/weather.html]: sure, will remember that.

**Pull request merged**